Sources/ATTNParameterValidation.m

```diff
@@ -31,8 +39,12 @@ + (void)verifyNotNil:(nonnull NSObject *)inputValue inputName:(nonnull const NSS
 }
 
 + (void)verifyStringOrNil:(nullable NSString *)inputValue inputName:(nonnull const NSString *)inputName {
-  if ([ATTNParameterValidation isNotNil:inputValue] && ![ATTNParameterValidation isStringAndNotEmpty:inputValue]) {
-    [NSException raise:@"Bad Identifier" format:@"%@ should be a non-empty NSString", inputName];
+  if ([ATTNParameterValidation isNotNil:inputValue]) {
+    if (![ATTNParameterValidation isString:inputValue]) {
+      [NSException raise:@"Bad Identifier" format:@"%@ should be NSString", inputName];
+    } else if ([ATTNParameterValidation isEmpty:inputValue]) {
+      NSLog(@"Identifier %@ should be non-empty", inputName);
+    }
```

Comment:
Now that I think about it, I want a separate method instead of rewriting this one.